active_plugins/runcellpose.py

```diff
@@ -300,16 +326,18 @@ def settings(self):
             self.stitch_threshold,
             self.do_3D,
             self.min_size,
-            self.omni,
+            # self.omni,

```

Comment:
Is it necessary? I know it won't run in Docker, presumably, but for folks running in Python, it would be nice to not remove

Reply:
Could just add a warning that for now, Omnipose has no available Dockers